modules/oe_bootstrap_theme_paragraphs/src/EventSubscriber/IconOptionsAlterSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_bootstrap_theme_paragraphs\EventSubscriber;
+
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides options for the icon field.
+ */
+class IconOptionsAlterSubscriber implements EventSubscriberInterface {
```

Comment:
we should name it the way it makes most sense, which for me is IconOptionsSubscriber, no need to find a different name just to differentiate that's what we have namespaces for